src/features/transfer-ownership/TransferOwnership.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { isValidEthAddress } from '../../lib/util/address/address';
+
+export const isValid = (value: string, account: string) =>
+	isValidEthAddress(value) && !(value.toLowerCase() === account.toLowerCase());
```

Comment:
Can we change the name of `isValid` to something more explicit? It looks like it's checking that the address is valid to send to, so perhaps we could change the function name to `isValidTransferAddress` or similar.

Reply:
Sure makes sense :)